examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -0,0 +1,111 @@
+"""
+===============================
+Compute and visualize ERDS maps
+===============================
+
+This example calculates and displays ERDS maps of event-related EEG data. ERDS
+(sometimes also written as ERD/ERS) is short for event-related
+desynchronization (ERD) and event-related synchronization (ERS) [1]_.
+Conceptually, ERD corresponds to a decrease in power in a specific frequency
+band relative to a baseline. Similarly, ERS corresponds to an increase in
+power. An ERDS map is a time/frequency representation of ERD/ERS over a range
+of frequencies [2]_.
+
+We use a public EEG BCI data set containing two different motor imagery tasks
+available at PhysioNet. The two tasks are imagined hand and feet movement. Our
+goal is to generate ERDS maps for each of the two tasks.
+
+First, we load the data and create epochs of 5s length. The data sets contain
+multiple channels, but we will only consider the three channels C3, Cz, and C4.
+We compute maps containing frequencies ranging from 2 to 30Hz. We map ERD to
+red color and ERS to blue color, which is the convention in many ERDS
+publications. Note that we do not perform any significance tests on the map
+values, but instead we display the whole time/frequency maps.
+
+References
+----------
+
+.. [1] G. Pfurtscheller, F. H. Lopes da Silva. Event-related EEG/MEG
+       synchronization and desynchronization: basic principles. Clinical
+       Neurophysiology 110(11), 1842-1857, 1999.
+.. [2] B. Graimann, J. E. Huggins, S. P. Levine, G. Pfurtscheller.
+       Visualization of significant ERD/ERS patterns in multichannel EEG and
+       ECoG data. Clinical  Neurophysiology 113(1), 43-47, 2002.
+"""
+# Authors: Clemens Brunner <clemens.brunner@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import LinearSegmentedColormap
+import mne
+from mne.datasets import eegbci
+from mne.io import concatenate_raws, read_raw_edf
+from mne.time_frequency import tfr_multitaper
+
+
+def center_cmap(cmap, vmin, vmax):
+    """Center given colormap at value 0.
+    """
+    vzero = abs(vmin) / (vmax - vmin)
+    index_old = np.linspace(0, 1, cmap.N)
+    index_new = np.hstack([np.linspace(0, vzero, cmap.N // 2, endpoint=False),
+                           np.linspace(vzero, 1, cmap.N // 2)])
+    cdict = {"red": [], "green": [], "blue": [], "alpha": []}
+    for old, new in zip(index_old, index_new):
+        r, g, b, a = cmap(old)
+        cdict["red"].append((new, r, r))
+        cdict["green"].append((new, g, g))
+        cdict["blue"].append((new, b, b))
+        cdict["alpha"].append((new, a, a))
+    return LinearSegmentedColormap("erds", cdict)
```

Comment:
This solution seems less preferable than going the `Norm` class route. Why not use the norm class that we discussed and use `pc.set_norm(...)`?

Reply:
Yes, I just had this example almost implemented with this quick fix. I'll try the other approach and see if it works.